packages/contracts/src/deployConfigs.ts

```diff
@@ -52,6 +52,29 @@ async function networkConfigs(chainId: number): Promise<Config> {
       };
     }
 
+    case "mandala":
+      const date = new Date();
+      const beginningOfNextMonth = new Date(
+        date.getFullYear(),
+        date.getMonth() + 1,
+        1,
+        12
+      );
+      const now = Math.floor(date.getTime() / 1000);
+      const oneDay = 60 * 60 * 24;
+
+      return {
+        ctndSale1: {
+          start: now,
+          end: now + oneDay * 7,
+          supply: parseUnits("10"),
+        },
+        ctndVesting: {
+          start: beginningOfNextMonth.getTime() / 1000,
+        },
+        ctndSale2: undefined,
```

Comment:
Why undefined here? Just want to test a sale and ignore the other, is that it?

Reply:
in this case I wanted to deploy sale1, but ignore sale2